packages/apps-config/src/settings/endpoints.ts

```diff
@@ -232,6 +232,11 @@ function createTestNetworks (t: TFunction): LinkOption[] {
       info: 'westend',
       text: t<string>('rpc.westend', 'Westend (Polkadot Testnet, hosted by Parity)', { ns: 'apps-config' }),
       value: 'wss://westend-rpc.polkadot.io'
+    },
```

Comment:
Alphabetical here as well, so move it up the list.

Reply:
Sure.